include/rtc/peerconnection.hpp

```diff
@@ -35,6 +35,12 @@ struct RTC_CPP_EXPORT DataChannelInit {
 	string protocol = "";
 };
 
+struct RTC_CPP_EXPORT LocalDescriptionInit {
+    Description::Type type = Description::Type::Unspec;
```

Comment:
You don't need this here if `LocalDescriptionInit` is a second parameter to `setLocalDescription()`.

Reply:
Good point, was from an earlier iteration.

Done!